recipes/num2words/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "num2words" %}
+{% set version = "0.5.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37cd4f60678f7e1045cdc3adf6acf93c8b41bf732da860f97d301f04e611cc57
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3
```

Comment:
It looks like this package supports `python >=2.7` so change this here and under `run`.

Reply:
Previous maintainers have said to make this pin be >=3 despite 2.7 capabilities. Here: https://github.com/conda-forge/staged-recipes/pull/13369#discussion_r533636576

Reply:
I'm not sure if there is a general consensus. I've been telling people that when the library supports it they should use `>=2.7` so that in principle people could use it in an older py2 environment if it is `noarch` and the environment will solve.

Reply:
Okay, going with that.